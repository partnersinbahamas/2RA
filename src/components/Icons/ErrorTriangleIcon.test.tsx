import React from 'react';
import { render } from '@testing-library/react';

import ErrorTriangleIcon from './ErrorTriangleIcon';

describe('ErrorTriangleIcon', () => {
  it('renders solid', () => {
    const { container } = render(<ErrorTriangleIcon size="SOLID" />);
    expect(container).toMatchSnapshot();
  });
  it('renders small', () => {
    const { container } = render(<ErrorTriangleIcon size="SMALL" />);
    expect(container).toMatchSnapshot();
  });
  it('renders medium', () => {
    const { container } = render(<ErrorTriangleIcon size="MEDIUM" />);
    expect(container).toMatchSnapshot();
  });
  it('renders large', () => {
    const { container } = render(<ErrorTriangleIcon size="LARGE" />);
    expect(container).toMatchSnapshot();
  });
  it('renders default', () => {
    const { container } = render(<ErrorTriangleIcon />);
    expect(container).toMatchSnapshot();
  });
});
