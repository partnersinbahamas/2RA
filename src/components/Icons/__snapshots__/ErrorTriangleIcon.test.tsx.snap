// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorTriangleIcon renders default 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="0 0 28 27"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M14 1.01268e-05C15.132 1.01268e-05 16.17 0.61201 16.71 1.61501L27.653 22.548C27.897 23 28 23.483 28 24C28 25.735 26.73 27 25 27H3.143C1.41 27 0 25.73 0 24C0 23.483 0.103 23 0.347 22.548L11.29 1.61501C11.5531 1.12565 11.9442 0.716941 12.4215 0.432504C12.8988 0.148067 13.4444 -0.00141722 14 1.01268e-05ZM14 1.84301C13.644 1.84301 13.354 2.10201 13.094 2.58501L2.104 23.485C2.01934 23.6313 1.97163 23.7961 1.965 23.965C1.965 24.397 2.353 25 3 25L24.965 25.035C25.518 25.035 26 24.52 26 24C25.9937 23.8312 25.9463 23.6664 25.862 23.52L14.906 2.58501C14.646 2.10201 14.356 1.84301 14 1.84301ZM12.61 21.596C12.61 22.34 13.256 22.986 14 22.986C14.3678 22.9834 14.7199 22.8361 14.98 22.576C15.2401 22.3159 15.3874 21.9639 15.39 21.596C15.3913 21.4131 15.3563 21.2318 15.2869 21.0625C15.2175 20.8933 15.1152 20.7395 14.9858 20.6102C14.8565 20.4808 14.7027 20.3785 14.5335 20.3091C14.3643 20.2397 14.1829 20.2047 14 20.206C13.6322 20.2086 13.2801 20.3559 13.02 20.616C12.7599 20.8761 12.6126 21.2282 12.61 21.596ZM12.965 17.166V8.01801C12.965 7.43601 13.418 6.95101 14 6.95101C14.582 6.95101 15.035 7.43701 15.035 8.01801V17.166C15.035 17.748 14.582 18.233 14 18.233C13.418 18.233 12.965 17.748 12.965 17.166Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`ErrorTriangleIcon renders large 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="0 0 28 27"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M14 1.01268e-05C15.132 1.01268e-05 16.17 0.61201 16.71 1.61501L27.653 22.548C27.897 23 28 23.483 28 24C28 25.735 26.73 27 25 27H3.143C1.41 27 0 25.73 0 24C0 23.483 0.103 23 0.347 22.548L11.29 1.61501C11.5531 1.12565 11.9442 0.716941 12.4215 0.432504C12.8988 0.148067 13.4444 -0.00141722 14 1.01268e-05ZM14 1.84301C13.644 1.84301 13.354 2.10201 13.094 2.58501L2.104 23.485C2.01934 23.6313 1.97163 23.7961 1.965 23.965C1.965 24.397 2.353 25 3 25L24.965 25.035C25.518 25.035 26 24.52 26 24C25.9937 23.8312 25.9463 23.6664 25.862 23.52L14.906 2.58501C14.646 2.10201 14.356 1.84301 14 1.84301ZM12.61 21.596C12.61 22.34 13.256 22.986 14 22.986C14.3678 22.9834 14.7199 22.8361 14.98 22.576C15.2401 22.3159 15.3874 21.9639 15.39 21.596C15.3913 21.4131 15.3563 21.2318 15.2869 21.0625C15.2175 20.8933 15.1152 20.7395 14.9858 20.6102C14.8565 20.4808 14.7027 20.3785 14.5335 20.3091C14.3643 20.2397 14.1829 20.2047 14 20.206C13.6322 20.2086 13.2801 20.3559 13.02 20.616C12.7599 20.8761 12.6126 21.2282 12.61 21.596ZM12.965 17.166V8.01801C12.965 7.43601 13.418 6.95101 14 6.95101C14.582 6.95101 15.035 7.43701 15.035 8.01801V17.166C15.035 17.748 14.582 18.233 14 18.233C13.418 18.233 12.965 17.748 12.965 17.166Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`ErrorTriangleIcon renders medium 1`] = `
<div>
  <svg
    class="sc-dmyCSP dRcSrL"
    fill="currentcolor"
    viewBox="0 0 20 20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M9.014 12.557V7.927C9.014 7.337 9.484 6.877 10.047 6.877C10.579 6.877 11.017 7.337 11.017 7.927V12.557C11.017 13.213 10.61 13.672 10.047 13.672C9.484 13.672 9.014 13.18 9.014 12.557ZM8.31 1.213L0.25 17.213C0.094 17.541 0 17.869 0 18.197C0 19.18 0.751 20 1.847 20H18.153C19.25 20 20 19.18 20 18.197C20 17.902 19.906 17.541 19.75 17.213L11.721 1.213C11.3 0.426 10.657 0 10 0C9.343 0 8.74 0.426 8.31 1.213ZM10.255 2.206L17.951 17.519C18.078 17.759 17.951 18 17.663 18H2.32C2.128 18 2 17.863 2 17.725C2 17.622 2.032 17.588 2.032 17.519L9.745 2.206C9.809 2.103 9.905 2 10 2C10.096 2 10.223 2.103 10.255 2.206ZM9.985 16.82C10.548 16.82 10.985 16.36 10.985 15.77C10.985 15.18 10.548 14.721 9.985 14.721C9.421 14.721 8.983 15.181 8.983 15.771C8.983 16.361 9.421 16.82 9.984 16.82H9.985Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`ErrorTriangleIcon renders small 1`] = `
<div>
  <svg
    class="sc-guDLey kBBjNb"
    fill="currentcolor"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M8 0C8.772 0 9.397 0.436 9.732 1.074L15.639 13.122C15.773 13.424 15.84 13.726 15.84 13.994C15.84 15.102 15.074 16 14 16H2C0.926 16 0 15.102 0 13.994C0 13.692 0.067 13.39 0.201 13.122L6.175 1.14C6.477 0.47 7.262 0 8 0ZM8 2L2 14H14L8 2ZM7 10V6H9V10H7ZM8 11.007C8.548 11.007 9 11.459 9 12.007C9 12.556 8.548 13.007 8 13.007C7.452 13.007 7 12.556 7 12.007C7 11.459 7.452 11.007 8 11.007Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`ErrorTriangleIcon renders solid 1`] = `
<div>
  <svg
    class="sc-guDLey kBBjNb"
    fill="currentcolor"
    viewBox="0 0 16 16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M7.999 0C8.794 0 9.493 0.422 9.815 1.07L15.793 13.111C15.934 13.395 15.997 13.697 15.997 13.98C15.997 15.09 15.079 16 13.977 16H2.02C0.918 16 0 15.09 0 13.98C0 13.677 0.07 13.378 0.204 13.111L6.224 1.111C6.38876 0.780451 6.64158 0.501805 6.9546 0.305772C7.26762 0.109738 7.62868 0.00392741 7.998 0H7.999ZM8 11.94C7.7259 11.94 7.46287 12.0482 7.26812 12.2411C7.07337 12.4339 6.96264 12.6959 6.96 12.97C6.96 13.246 7.06962 13.5106 7.26476 13.7057C7.45989 13.9009 7.72454 14.0105 8.0005 14.0105C8.27646 14.0105 8.54111 13.9009 8.73624 13.7057C8.93138 13.5106 9.041 13.246 9.041 12.97C9.041 12.404 8.571 11.94 8 11.94ZM9 4H7V10H9V4Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
